lib/rake/build.rake

```diff
@@ -56,14 +56,13 @@ namespace :build do
     end
   end
 
+  # TODO: (brent) - temporarily leave in a build step that just does a clean of
+  # code-studio to make sure we don't have artifacts from old builds
```

Comment:
I believe this task only runs if build_code_studio is true - and it only affects stuff within the code-studio directory, which in turn will only have an effect if use_my_code_studio is true, or you're doing the S3 packaging (in which case it makes more sense to make the fixes there - not sure what it'll do if the build directory is just missing).